compiler/src/dotty/tools/dotc/core/Denotations.scala

```diff
@@ -277,7 +277,7 @@ object Denotations {
       disambiguate(p) match {
         case m @ MissingRef(ownerd, name) =>
           if (generateStubs) {
-            m.ex.printStackTrace()
+            if (ctx.settings.YdebugMissingRefs.value) m.ex.printStackTrace()
```

Comment:
This check should also be done at https://github.com/lampepfl/dotty/blob/67e34d7674f03b48c863322bba8ac02893d0b554/compiler/src/dotty/tools/dotc/core/unpickleScala2/Scala2Unpickler.scala#L412 instead of ctx.debug